src/main/java/slimeknights/tconstruct/tools/data/MaterialStats.java

```diff
@@ -14,137 +14,156 @@
 
 final class MaterialStats {
 
-  private static final int STONE = 0;
-  private static final int IRON = 1;
-  private static final int DIAMOND = 2;
-  private static final int OBSIDIAN = 3;
-  private static final int COBALT = 4;
+  private static final int WOOD = 0;
+  private static final int STONE = 1;
+  private static final int IRON = 2;
+  private static final int DIAMOND = 3;
+  private static final int NETHERITE = 4;
+  private static final int MANYULLYN = 5;
 
   static final Map<MaterialId, List<IMaterialStats>> allMaterialStats = new HashMap<>();
 
   static {
+    //Tier 1
     addMaterialStats(MaterialIds.wood,
-        new HeadMaterialStats(35, 2.00f, STONE, 2.00f),
-        new HandleMaterialStats(1.00f, 25),
-        new ExtraMaterialStats(15));
-
-    addMaterialStats(MaterialIds.stone,
-        new HeadMaterialStats(120, 4.00f, IRON, 3.00f),
-        new HandleMaterialStats(0.50f, -50),
-        new ExtraMaterialStats(20));
-    addMaterialStats(MaterialIds.flint,
-        new HeadMaterialStats(150, 5.00f, IRON, 2.90f),
-        new HandleMaterialStats(0.60f, -60),
-        new ExtraMaterialStats(40));
-    addMaterialStats(MaterialIds.cactus,
-        new HeadMaterialStats(210, 4.00f, IRON, 3.40f),
-        new HandleMaterialStats(0.85f, 20),
-        new ExtraMaterialStats(50));
+      new HeadMaterialStats(60, 2f, WOOD, 0f),
+      new HandleMaterialStats(0, 1.0f, 1.0f, 1.0f),
+      new ExtraMaterialStats(0));
```

Comment:
I thought we were removing the durability from `Extra`? I'd rather not have it at all than have it and always set it to 0.

If we have to have a stat, give it a multiplier or a bonus to something not covered by handles to keep math simplier (no questions of if extra applies before or after handle)

Reply:
Woops, missed that one entirely.